docs/en/stack/ml/df-analytics/flightdata-classification.asciidoc

```diff
@@ -0,0 +1,477 @@
+[role="xpack"]
+[testenv="platinum"]
+[[flightdata-classification]]
+=== Predicting delaying flights with {classanalysis}
+
+Let's try to predict whether a flight will be delayed or not by using the 
+{kibana-ref}/add-sample-data.html[sample flight data]. We use this data to learn 
```

Comment:
> We use this data to learn a classification model....

In my opinion, it might be too soon to talk about the model.  I'd suggest something more high-level like this:

For example, we want to be able to use information such as weather conditions, carrier, flight distance, origin, or destination to predict flight delays.